README.md

```diff
@@ -167,7 +167,7 @@ In order to test your feature branch on staging you need to
 3. Uninstall current superset version
 
 ```shell
-$ pip uninstall hq-superset
+$ pip uninstall commcare-analytics
```

Comment:
I thought this was because the package was still named hq-superset, no?

Reply:
okay thank you.
I wonder old installations with previous name will continue to exist but that is probably hardly anyone.